target/linux/realtek/image/Makefile

```diff
@@ -3,7 +3,7 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
-KERNEL_LOADADDR = 0x80100000
+KERNEL_LOADADDR := 0x80100000
```

Comment:
Do you need to change the assignment type here for any reason? Without variables in the assignement, the `:=` assignment would only serve to make things marginally more efficient AFAICT (not re-evaluating the expression for every use).

Reply:
Simply following the standard

```
[markus@openwrt linux]$ grep KERNEL_LOADADDR */image/Makefile | grep " := " | wc -l
18
[markus@openwrt linux]$ grep KERNEL_LOADADDR */image/Makefile | grep " = " | wc -l
4

```

Reply:
Fine with that?